serviceinsight/sequence-diagram/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Sequence Diagram
+summary: ServiceInsight Sequence Diagram view
+tags: 
+- ServiceInsight
+- Sequence Diagram
+---
+
+# Introduction
+
+Each visualization diagram in ServiceInsight displays the currently selected selected message in a broader context to help with troubleshooting and debugging.
+
+The sequence diagram shows all of the messages in the same conversation. It also shows when these messages were sent and handled relative to each other. 
+
+!Image of Sequence Diagram
+
+Use the sequence diagram when the timing of when messages were handled is important.
+
+## Which data is used to generate the diagram
+
+When you send a message through NServiceBus, some headers are added automatically. All messages get the `NServiceBus.MessageId` header. This is a unqiue identifier for the message. If handling a message causes more messages to be sent or published then the `MessageId` of the message being handled is copied to the `NServiceBus.RelatedTo` header of the outgoing messages.
+
+
+## What is on the diagram
+
+The sequence diagram shows a complex interraction of endpoints sending and processing messages over time. 
```

Comment:
Emphasise it shows data at runtime level (e.g. Ramon wasn't sure about whether it was runtime or design data).